modules/tensor_mechanics/src/actions/CavityPressureAction.C

```diff
@@ -0,0 +1,105 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "CavityPressureAction.h"
+
+#include "Factory.h"
+#include "FEProblem.h"
+#include "Parser.h"
+#include "Conversion.h"
+
+template<>
+InputParameters validParams<CavityPressureAction>()
+{
+  InputParameters params = validParams<Action>();
+  params.addRequiredParam<std::vector<BoundaryName> >("boundary", "The list of boundary IDs from the mesh where the pressure will be applied");
+  params.addParam<std::vector<NonlinearVariableName> >("displacements", "The nonlinear displacement variables");
+  params.addParam<NonlinearVariableName>("disp_x", "The x displacement");//deprecated
+  params.addParam<NonlinearVariableName>("disp_y", "The y displacement");//deprecated
+  params.addParam<NonlinearVariableName>("disp_z", "The z displacement");//deprecated
```

Comment:
Remove the old syntax will break all the corresponding BISON tests. So, for this purpose, I just keep it as deprecated.